src/components/common/TmSessionExtension.vue

```diff
@@ -62,15 +62,6 @@ export default {
       return this.extension.accounts
     }
   },
-  async created() {
-    if (this.$route.params.address && this.$route.params.network) {
-      await this.signIn({
-        address: this.$route.params.address,
-        network: this.$route.params.network
-      })
-      this.$router.push("/portfolio")
-    }
-  },
```

Comment:
Shouldn't this be another PR 😛 ?

Reply:
left overs, I can clean it up if necessary

Reply:
Please, split PRs to make more readable & understandable💃

Reply:
sure when I clean this up